src/Microsoft.Identity.Web/TokenAcquisitionOptions.cs

```diff
@@ -46,6 +47,11 @@ public class TokenAcquisitionOptions
         /// </summary>
         public PoPAuthenticationConfiguration? PoPConfiguration { get; set; }
 
+        /// <summary>
+        /// todo.
+        /// </summary>
+        public CancellationToken CancellationToken { get; set; } = CancellationToken.None;
```

Comment:
For a few minutes, I was wondering if we would want to use `CacheSerializerHints` instead of `CancellationToken`
but thinking twice it would not make sense as the properties of the Hint will come from MSAL.NET Token serialization args, not necessarily the developers.

So nothing to change here, but I thought I would explain my thoughts.